demos/keyboard-a11y/js/five-star.js

```diff
@@ -45,6 +45,7 @@ var demo = demo || {};
             "role": "radio",
             "aria-checked": false
         });
+        demo.fiveStar.updateARIA(stars,1);
```

Comment:
What if the model is set to something other than 1? It's probably better to investigate why the modelListener is not setting the value correctly. Additionally, you'll want to ensure that the `tabindex` is set to 0 for the initial star as well. You may need to look into how the items are selected when setting a different rank.